scripts/performance/perf_processes.py

```diff
@@ -302,8 +317,13 @@ def _rand_data(self):
 
     def _from_file_str_data(self, file_str):
         req_json = super()._from_file_str_data(file_str)
-        req_did = json.loads(req_json)['result']['txn']['data']['dest']
-        return req_did
+        if req_json is None:
+            return None
+        tmp = json.loads(req_json)
+        txn = tmp.get('result', {}).get('txn', {}) or tmp.get('txn', {})
```

Comment:
This looks like duplicate code in multiple places. Can we use some utility/helper method?